test/e2e-cert-rotation-disruptive/pki.go

```
@@ -0,0 +1,740 @@
+		var err error
+		kubeClient, err = getKubeClient()
+		if err != nil {
+			g.Fail(fmt.Sprintf("error getting kube client: %v", err))
+		}
+
+		configClient, err = getConfigClient()
+		if err != nil {
+			g.Fail(fmt.Sprintf("error getting config client: %v", err))
+		}
+
+		ctx, cancel = context.WithTimeout(context.Background(), 2*time.Hour)
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)